cwms-data-api/src/main/java/cwms/cda/data/dto/Catalog.java

```diff
@@ -95,55 +112,90 @@ public CatalogPage(String page){
             this.cursorId = parts[1];
             this.searchOffice = office;
-            this.idLike = idLike;                
+            this.idLike = idLike;
             this.locCategoryLike = locCategoryLike;
             this.locGroupLike = locGroupLike;
             this.tsCategoryLike = tsCategoryLike;
             this.tsGroupLike = tsGroupLike;
+            this.boundingOfficeLike = boundingLike;
         }
 
         private String nullOrVal(String val) {
-            if( val == null || val.equalsIgnoreCase("null")) {
+            if (val == null || val.equalsIgnoreCase("null")) {
                 return null;
             } else {
                 return val;
             }
         }
-        
-        public String getSearchOffice(){ return searchOffice; }
-        public String getCurOffice(){ return curOffice; }
-        public String getCursorId() { return cursorId; }
-        public int getPageSize() { return pageSize; }
-        public int getTotal() { return total; }
-        public String getIdLike() { return idLike; }
-        public String getLocCategoryLike() { return locCategoryLike; }
-        public String getLocGroupLike() { return locGroupLike; }
-        public String getTsCategoryLike() { return tsCategoryLike; }
-        public String getTsGroupLike() { return tsGroupLike; }
+
+        public String getSearchOffice() {
+            return searchOffice;
+        }
+
+        public String getCurOffice() {
+            return curOffice;
+        }
+
+        public String getCursorId() {
+            return cursorId;
+        }
+
+        public int getPageSize() {
+            return pageSize;
+        }
+
+        public int getTotal() {
+            return total;
+        }
+
+        public String getIdLike() {
+            return idLike;
+        }
+
+        public String getLocCategoryLike() {
+            return locCategoryLike;
+        }
+
+        public String getLocGroupLike() {
+            return locGroupLike;
+        }
+
+        public String getTsCategoryLike() {
+            return tsCategoryLike;
+        }
+
+        public String getTsGroupLike() {
+            return tsGroupLike;
+        }
+
+        public String getBoundingOfficeLike() {
+            return boundingOfficeLike;
+        }
 
         @Override
         public String toString() {
-            StringBuilder sb = new StringBuilder();
-            sb.append(curOffice).append("/").append(cursorId)
-              .append(CwmsDTOPaginated.delimiter).append(searchOffice)
-              .append(CwmsDTOPaginated.delimiter).append(idLike)
-              .append(CwmsDTOPaginated.delimiter).append(locCategoryLike)
-              .append(CwmsDTOPaginated.delimiter).append(locGroupLike)
-              .append(CwmsDTOPaginated.delimiter).append(tsCategoryLike)
-              .append(CwmsDTOPaginated.delimiter).append(tsGroupLike);
-            return sb.toString();
+            return curOffice + "/" + cursorId
```

Comment:
This is cleaner looking though I wonder if in the future we might want to create a "cursor builder". the delimited section would lend itself fairly easily to a `.items(String delimeter, String ...items)` type call vs doing the `+` everywhere.